src/ui/components/EntryContextMenu/constants.ts

```diff
@@ -38,7 +38,11 @@ const CONTEXT_MENU_COPY = {
             : undefined;
     },
     // remain Copy menu item in navigation, but show in actionPanel only if actual version is opened
-    isVisible({showSpecificItems}: ContextMenuParams) {
+    isVisible({showSpecificItems, entry}: ContextMenuParams) {
+        if (entry?.workbookId) {
```

Comment:
It also hides "Duplicate" button on workbook entity page.

The action should remain on workbook entity page:
<img width="622" alt="Screenshot 2024-04-25 at 12 46 14" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/4e7b5ff1-e94f-4ae6-8c20-1065db0de557">

Reply:
added flag - isFlatList for hide only in flat list place